src/SynthGrid.cpp

```diff
@@ -938,16 +938,21 @@ void SynthGrid::copySynth()
 
 void SynthGrid::pasteSynth()
 {
-	ModularSynth& synth = getModularSynth();
-	synth.lock();
-	synth.copy(*mCopyBuffer);
-	synth.unlock();
-
+	if (mCopyBuffer != NULL)
+	{
+		ModularSynth& synth = getModularSynth();
+		synth.lock();
+		synth.copy(*mCopyBuffer);
+		synth.unlock();
+	
 	rebuildWires();
-
+	
+	refreshView();
```

Comment:
Can you please indent this properly before merge? Alternatively, you can just do 
```
if (mCopyBuffer == NULL)
{
  return;
}
```

very first in the method and leave the rest untouched.